build/rules/checkstyle_rules.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
can we rename this folder as `checkstyle` and keep the file under it

Reply:
Yeah. Sure